Common/Data/Dialogs/dlgConfiguration_L.xml

```diff
@@ -1,533 +1,525 @@
-      <WndProperty Name="prpEngineeringMenu" Caption="_@M260_"  X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H213_">
-        <DataField Name="" DataType="boolean"/>
-      </WndProperty>
-    </WndFrame>
-    <WndFrame Name="frmSpecials2" X="70" Y="0" Width="-1" Height="222" Font="2">
-      <WndProperty Name="prpPGClimbZoom" Caption="_@M170_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H214_">
-        <DataField Name="" DataType="enum"  Min="0" Max="2" Step="1"/>
-      </WndProperty>
-      <WndProperty Name="prpPGCruiseZoom" Caption="_@M202_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H215_">
-        <DataField Name="" DataType="enum"  Min="0" Max="2" Step="1"/>
-      </WndProperty>
-      <WndProperty Name="prpPGAutoZoomThreshold" Caption="_@M1697_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H1213_">
-        <DataField Name="" DataType="double" DisplayFormat="%.1f %s" EditFormat="%.1f" Min="1.0" Max="20.0" Step="0.5" />
-      </WndProperty>
-      <WndProperty Name="prpUseUngestures" Caption="_@M1792_"  X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H1260_">
-        <DataField Name="" DataType="boolean"/>
-      </WndProperty>
-      <WndProperty Name="prpPGOptimizeRoute" Caption="_@M1686_"  X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H9999_">
-        <DataField Name="" DataType="boolean"/>
-      </WndProperty>
-    </WndFrame>
-    <WndFrame Name="frmEngineering1" X="70" Y="0" Width="-1" Height="222" Font="1">
-      <WndProperty Name="prpDebounceTimeout" Caption="_@M220_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H221_">
-        <DataField Name="" DataType="double" DisplayFormat="%.0f ms" EditFormat="%.0f" Min="0" Max="1000" Step="25"/>
-      </WndProperty>
-      <WndProperty Name="prpInputFile" Caption="_@M268_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="130" Font="2" Help="_@H222_">
-        <DataField Name="" DataType="filereader" Min="0" Max="200" Step="1"/>
-      </WndProperty>
-     <WndProperty Name="prpCheckSum" Caption="_@M449_" X="2" Y="-1" Width="-1" Height="21" CaptionWidth="150" Font="2" Help="_@H161_">
-        <DataField Name="" DataType="enum"  Min="0" Max="1" Step="1"/>
-      </WndProperty>
-      <WndProperty Name="prpIphoneGestures" Caption="_@M346_" X="2" Y="-1"  Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H180_">
-        <DataField Name="" DataType="enum" Min="0" Max="1" Step="1" />
-      </WndProperty>
-      <WndProperty Name="prpGPSAltitudeOffset" Caption="_@M318_" X="2" Y="-1" Width="-1" Height="22" CaptionWidth="150" Font="2" Help="_@H159_" Keyboard="1">
-        <DataField Name="" DataType="double" DisplayFormat="%.0f %s" EditFormat="%.0f" Min="-1000" Max="1000" Step="1"/>
-      </WndProperty>
-    </WndFrame>
-  </WndForm>
-</PMML>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It will be difficult here to resolve conflicts. I think something is wrong with style.

Reply:
Did I?
Sorry, seems to be removed accidentally!

Reply:
Can't find a problem in Commit.

Reply:
No problem. It looks like the entire file has been modified. Maybe some <CR><LF> ?  As this will certainly conflict with my upcoming PR I was just wandering what has been changed. No problem anyway .. I can look at the _P version that looks ok.